dali/pipeline/operators/reader/parser/tfrecord_parser.h

```diff
@@ -97,7 +110,19 @@ class TFRecordParser : public Parser<Tensor<CPUBackend>> {
           break;
         case FeatureType::float32:
           if (!f.HasShape()) {
-            output.Resize({encoded_feature.float_list().value().size()});
+            if (f.HasPartialShape()) {
```

Comment:
wondering, can this duplicated code be extracted to a templated function?

Reply:
Done